### ISSUE TITLE
Metadata report using CAAIS consultation draft

### ISSUE THREAD

**Issue by whikloj** [NationalCentreTruthReconciliation/Secure-Record-Transfer#73]
It looks like section 4.3 changed between the [Consultation Draft](http://archivescanada.ca/uploads/files/Documents/CAAIS_ConsultationDraft_Final.pdf) and the [Final version](http://archivescanada.ca/uploads/files/Documents/CAAIS_2019May15_EN.pdf).

```
4.3 Material Assessment Statement............................................................... 23
4.3.1 Material Assessment Statement Type ....................................................... 24
4.3.2 Material Assessment Statement Value....................................................... 24
4.3.3 Material Assessment Action Plan .......................................................... 25
4.3.4 Material Assessment Statement Note ....................................................... 25
```
```
4.3 Preservation Requirements....................................................................32
     4.3.1 Preservation Requirement Type ........................................................33
     4.3.2 Preservation Requirement Value .......................................................34
     4.3.3 Preservation Requirement Note.........................................................34
```

Minimally the metadata report should be updated.
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/master/bagitobjecttransfer/recordtransfer/templates/recordtransfer/report/metadata_report.html#L307

**Comment by danloveg** [comment]
Thanks for flagging this. The metadata report _and_ `caais.py` will need to be changed. This also influences #80, as the contact assessment will now go under 4.3 Preservation Requirements. I will merge in #80 and make the changes required to reflect the updated section name.

I will plan to also make the template backward compatible with the old 4.3 Material Assessment Statement.

**Comment by danloveg** [comment]
Closed by #153
